Create security.md (#253)
solid/process: security.md

### Security Vulnerability Reporting 
The Product Security Incident Response Team (PSIRT) at Solid acknowledges the valuable role researchers play. We encourage reporting of any concerns and vulnerabilities found in our sites or software. 

> info@solidproject.org
> Submit an issue to our team on github

Reviewer: ```suggestion
In order to report any concern:
* Submit an issue to our team on github; or
* Email: info@solidproject.org
```

> info@solidproject.org
> Submit an issue to our team on github

Reviewer 2: I'm not sure which repo these issues belong on.
```suggestion
Please submit any issues to [our team on github](needs_the_repo/issues/), or email <info@solidproject.org>.
```

We are committed to working with the community to verify and respond to these reports in a timely fashion. Here's what you can expect when submitting a report:

* Acknowledgement of report receipt
* Communication of estimated time for resolution
* Notification of fix

We request that the following research not be conducted without formal authorization and advance coordination to avoid harms to customers and violation of laws. 

> * Denial of Service (DoS) of any kind
> * Automated security tools
> * Accessing, or attempting to access, data or information that does not belong to you
> * Destroying or corrupting, or attempting to destroy or corrupt, data or information that does not belong to you

Reviewer 2: ```suggestion
* Denial of Service (DoS) testing of any kind
* Automated security testing
* Testing access to data or information that does not belong to you
* Testing ability to destroy or corrupt data or information that does not belong to you
```

> Software often contains third party or open source libraries and binaries. Prior to submitting a request to validate how a security issue in third party components may impact Solid, please review the section on third party CVE handling.

Reviewer: ```suggestion
Software often contains third party or open source libraries and binaries. Prior to submitting a request to validate how a security issue in third party components may impact Solid, please review the section on third party Common Vulnerabilities and Exposures (CVE) handling.
```

> Software often contains third party or open source libraries and binaries. Prior to submitting a request to validate how a security issue in third party components may impact Solid, please review the section on third party CVE handling.

Reviewer 2: Software often contains third party or open source libraries and binaries. Prior to submitting a request to validate how a security issue in third party components may impact Solid, please review the section on Handling Third Party CVE (Common Vulnerabilities and Exposures).

> ### Third Party CVE Handling
> The Solid team updates 3rd party components within regularly scheduled release cycles, to the newest compatible version available during development. A vulnerability related to a 3rd party component does not necessarily translate to a vulnerability in Inrupt software. PSIRT welcomes questions about the applicability of a 3rd party CVE.

Reviewer 2: It *might* be appropriate to change "vulnerability in Solid software" to "vulnerability in Solid software from Inrupt", but it's not clear to me that this only about Inrupt.
```suggestion
### Handling Third Party CVE

The Solid team updates third party components to the newest compatible version available during development in regularly scheduled release cycles. A vulnerability related to a third party component does not necessarily translate to a vulnerability in Solid software. PSIRT welcomes questions about the applicability of a Third Party CVE.
```

Risk is determined through internal scoring using CVSSv3.1 (https://www.first.org/cvss/calculator/3.1). 

### Security Advisories 

> Notifications and descriptions of security incidents are available here.

Reviewer: To be clear, at loss of a better solution, are you recommended that the list of Security Advisories related to Solid will be currated manually as a list:
- inside this present document; or;
- in an external `security-advisories.md`?

I would probably have a slight preference for the latter.

> Notifications and descriptions of security incidents are available here.

Reviewer 2: ```suggestion
Notifications and descriptions of security incidents are available [here](needs_a_link_to_document_or_directory).
```

> Security Advisories and other security content are provided on an "as is" basis and do not imply any kind of guarantee or warranty. Your use of the information in these publications or linked material is at your own risk. Inrupt reserves the right to change or update this content without notice at any time.

Reviewer 2: Inrupt.com?  Or solidproject.org?  This document is starting to exhibit a split personality...
